backend/src/Wasm/Libs/Editor.fs

```diff
@@ -143,11 +143,14 @@ let fns : List<BuiltInFn> =
                 let inputVars = Map.empty
                 LibExecution.Execution.executeExpr state inputVars expr
 
-              return
-                LibExecution.DvalReprDeveloper.toRepr result
-                |> DString
-                |> Ok
-                |> DResult
+              match result with
+              | DError (_source, err) -> return DResult(Error(DString err))
+              | result ->
+                return
+                  LibExecution.DvalReprDeveloper.toRepr result
+                  |> DString
+                  |> Ok
+                  |> DResult
             with
```

Comment:
Is there a need for a try/catch block in this function? I don't believe there is a scenario where we would encounter an exception in this specific context. Or am I overlooking something?

Reply:
I think you're right! Can probably be removed.

Reply:
I'll remove it in the next PR (you merged before I got the chance to update it 🙂)